hnginternship5/hng.tech
# Create the HNG Finalists Page and integrate to the repo

### What does this PR do?

Creates the HNG 5 Finalists page

### How can this be tested?
>- Clone the repository
>- Run `npm install`
>- Navigate to the `views` directory and open `interns.php`. To see this page, click the 'See More' button on the 'Meet the Interns' section
